src/constraints/storage_constraints.jl

```diff
@@ -128,6 +128,19 @@ function add_elec_storage_dispatch_constraints(m, p, b; _n="")
     end
 end
 
+function add_elec_storage_cost_constant_constraints(m, p, b; _n="")
+    # Include the electric storage cost constants only if the installed_cost_constant or the replace_cost_constant is not zero
+    for b in p.s.storage.types.elec
```

Comment:
@toddleif since "b" is a function input, I don't think this function should include a for loop over b 's in storage.types.elec. Please correct me if I'm wrong

Reply:
That makes sense. I just removed the for loop.